lms/djangoapps/instructor_task/models.py

```diff
@@ -178,11 +178,11 @@ def create_output_for_failure(exception, traceback_string):
                     too_long -= (len(traceback_string) + len('traceback'))
                 else:
                     # truncate the traceback:
-                    task_progress['traceback'] = traceback_string[:-(too_long + len(tag))] + tag
+                    task_progress['traceback'] = traceback_string[(too_long + len(tag)):] + tag
```

Comment:
Now celery is sending complete traceback and by trimming from end it removes the exception. So removes the initial part instead.